README.md

```diff
@@ -14,48 +14,13 @@ Use [PEP 484](https://www.python.org/dev/peps/pep-0484/) type annotations plus [
 
 ## Dev
 
-### Plugin Builder
+After editing the plugin, just run `make` to build a zip with the plugin. It
+will be outputted in zip_buil/picterra.zip.
```

Comment:
Seems the zip is outputted in current directory

Reply:
I get it in zip_build:

```
Remove all compiled resource and ui files from the project? [y/N]: Cleaning resource and ui files
Deleted: dialog_detect.py
Deleted: dialog_results.py
Deleted: dialog_settings.py
Deleted: dialog_upload.py
Deleted: resources.py
pb_tool compile
Compiling dialog_detect.ui to dialog_detect.py
Compiling dialog_results.ui to dialog_results.py
Compiling dialog_settings.ui to dialog_settings.py
Compiling dialog_upload.ui to dialog_upload.py
Compiled 4 UI files
Compiling resources.qrc to resources.py
Compiled 1 resource files
pb_tool zip
Found zip: /usr/bin/zip
Deploying to zip_build/picterra
Removing plugin from zip_build/picterra
Plugin was not deleted: No such file or directory
Deploying to zip_build/picterra
Compiling to make sure install is clean
Skipping dialog_detect.ui (unchanged)
Skipping dialog_results.ui (unchanged)
Skipping dialog_settings.ui (unchanged)
Skipping dialog_upload.ui (unchanged)
Compiled 0 UI files
Skipping resources.qrc (unchanged)
Compiled 0 resource files
Building the help documentation
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v3.1.2
WARNING: html_static_path entry '_static' does not exist
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
no targets are out of date.
build succeeded, 1 warning.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
Copying __init__.py
Copying picterra.py
Copying dialog_detect.py
Copying dialog_results.py
Copying dialog_settings.py
Copying dialog_upload.py
Copying resources.py
Copying metadata.txt
Copying contents of assets to zip_build/picterra
Copying contents of i18n to zip_build/picterra
Copying contents of src to zip_build/picterra
Copying help/build/html to zip_build/picterra/help
  adding: picterra/ (stored 0%)
  adding: picterra/metadata.txt (deflated 51%)
  adding: picterra/dialog_results.py (deflated 79%)
  adding: picterra/__init__.py (deflated 63%)
  adding: picterra/dialog_upload.py (deflated 84%)
  adding: picterra/picterra.py (deflated 72%)
  adding: picterra/dialog_settings.py (deflated 82%)
  adding: picterra/dialog_detect.py (deflated 81%)
  adding: picterra/resources.py (deflated 68%)
  adding: picterra/i18n/ (stored 0%)
  adding: picterra/i18n/picterra.pro (deflated 43%)
  adding: picterra/i18n/picterra_fr.ts (deflated 86%)
  adding: picterra/i18n/picterra_it.ts (deflated 87%)
  adding: picterra/assets/ (stored 0%)
  adding: picterra/assets/info.png (stored 0%)
  adding: picterra/assets/add.png (stored 0%)
  adding: picterra/assets/upload2.png (stored 0%)
  adding: picterra/assets/warning.png (stored 0%)
  adding: picterra/assets/tabs.png (stored 0%)
  adding: picterra/assets/icon.png (stored 0%)
  adding: picterra/assets/ok.png (stored 0%)
  adding: picterra/assets/download.png (stored 0%)
  adding: picterra/assets/settings.png (stored 0%)
  adding: picterra/assets/spinner.gif (deflated 51%)
  adding: picterra/assets/detect.png (stored 0%)
  adding: picterra/assets/platform.png (stored 0%)
  adding: picterra/assets/help.png (stored 0%)
  adding: picterra/assets/error.png (stored 0%)
  adding: picterra/assets/upload.png (stored 0%)
  adding: picterra/help/ (stored 0%)
  adding: picterra/help/index.html (deflated 71%)
  adding: picterra/help/searchindex.js (deflated 53%)
  adding: picterra/help/_sources/ (stored 0%)
  adding: picterra/help/_sources/index.rst.txt (deflated 41%)
  adding: picterra/help/_static/ (stored 0%)
  adding: picterra/help/_static/plus.png (deflated 7%)
  adding: picterra/help/_static/underscore.js (deflated 66%)
  adding: picterra/help/_static/classic.css (deflated 73%)
  adding: picterra/help/_static/documentation_options.js (deflated 33%)
  adding: picterra/help/_static/searchtools.js (deflated 68%)
  adding: picterra/help/_static/file.png (stored 0%)
  adding: picterra/help/_static/sidebar.js (deflated 69%)
  adding: picterra/help/_static/jquery-3.5.1.js (deflated 71%)
  adding: picterra/help/_static/underscore-1.3.1.js (deflated 72%)
  adding: picterra/help/_static/language_data.js (deflated 65%)
  adding: picterra/help/_static/minus.png (deflated 8%)
  adding: picterra/help/_static/default.css (stored 0%)
  adding: picterra/help/_static/basic.css (deflated 77%)
  adding: picterra/help/_static/pygments.css (deflated 79%)
  adding: picterra/help/_static/doctools.js (deflated 64%)
  adding: picterra/help/_static/jquery.js (deflated 66%)
  adding: picterra/help/.buildinfo (deflated 25%)
  adding: picterra/help/genindex.html (deflated 69%)
  adding: picterra/help/search.html (deflated 68%)
  adding: picterra/help/objects.inv (stored 0%)
  adding: picterra/src/ (stored 0%)
  adding: picterra/src/__init__.py (stored 0%)
  adding: picterra/src/api.py (deflated 77%)
  adding: picterra/src/utils.py (deflated 68%)
  adding: picterra/src/network.py (deflated 74%)
  adding: picterra/src/dialogs.py (deflated 75%)
The picterra.zip archive has been created in the zip_build directory
The zip_build directory is temporary; don't store anything of value there.
```

Doesn't do that on your system (I'm on OSX) ?